molecule.yml

```diff
@@ -0,0 +1,16 @@
+---
+ansible:
+  config_file: ansible.cfg
+  sudo: True
```

Comment:
In order to install `azavea.unzip`, we'll probably need `requirements_file` pointing to `roles.yml`.